pkg/controller/datavolume-controller_test.go

```diff
@@ -1453,6 +1453,59 @@ var _ = Describe("All DataVolume Tests", func() {
 			Entry("snapshot", cdiv1.CDICloneStrategy(cdiv1.CloneStrategySnapshot)),
 		)
 
+		DescribeTable("After smart clone", func(actualSize resource.Quantity, currentSize resource.Quantity, expectedDvPhase cdiv1.DataVolumePhase) {
+			strategy := cdiv1.CDICloneStrategy(cdiv1.CloneStrategySnapshot)
```

Comment:
This line inspired me to write https://github.com/kubevirt/containerized-data-importer/pull/2277 there should be no reason that you need to cast the const to the right type.